tests/testthat/test-get-user-teams.R

```diff
@@ -0,0 +1,10 @@
+context("test-get-user-teams.R")
+
+library("synapser")
+if (on_travis()) syn_travis_login() else synLogin()
```

Comment:
I see that you have a synapse login in Travis.  Just out of curiosity, what would the code be to mock the synapse store function

Reply:
I have no idea honestly. I've done mocking before for simple HTTP responses, but I'm not sure what it would take to mock the synapser/PythonEmbedInR objects.